README.md

```diff
@@ -67,7 +67,14 @@ make docker-build
   </details>
 </div>
 
-6. Deploy.
+6. Prepare secret with Gardener credentials
```

Comment:
Could **Deploy** and **Prepare secret** steps be reverted in order? Deploy will then create a namespace needed by the other step.